### PR TITLE
update prezto integration instructions

### PR DESCRIPTION
**Pull request by sergiorussia**
updated instructions after https://github.com/sorin-ionescu/prezto/pull/1472

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,18 +63,18 @@ Note you must have the option PROMPT_SUBST set, see zshoptions(1).
...
 ---
 If you use [prezto](https://github.com/sorin-ionescu/prezto) you should do the following:
-
 ```
-cd ~/.zprezto/ \
-&& git submodule add https://github.com/miekg/lean.git modules/prompt/external/lean 2>/dev/null \
-&& git submodule update --init --recursive \
-&& cd modules/prompt/functions \
-&& ln -s ../external/lean/prompt_lean_setup
+cd $ZPREZTODIR && git clone --recurse-submodules https://github.com/belak/prezto-contrib contrib
```
Comment (miekg) [inline README.md:67]: does this dir and env var exist by default in prezto?

**Review by miekg — COMMENTED** [review]
Nice that *lean* is <strike>not</strike> *now* in prezto

File: README.md (hunk 1 above)
Comment (sergiorussia) [reply README.md:67]: env var yes. `contrib` dir is the default for additional zprezto modules, lean theme was added there (see https://github.com/belak/prezto-contrib/tree/master/contrib-prompt/external)

Comment (sergiorussia) [reply README.md:67]: btw i should update lean ref there...

Comment (miekg) [reply README.md:67]: ok, works for me

**Pull request merged**